### PR TITLE
custom have fun messages for team mails. ref #308

### PR DESCRIPTION
**Pull request by conradlempert**

### CODE REVIEW

**Review by birneamstiel — APPROVED** [review]

File: app/views/team_mailer/team_registered_to_event.html.erb [hunk 1]
```diff
@@ -2,5 +2,5 @@
 <p><%= t('.registered_to_event_info_html',
          team_link: link_to(@team.name, team_url(@team)),
          event_link: link_to(@event.name, event_url(@event))) %></p>
-<p><%= t('mailer.have_fun') %></p>
+<p><%= t('.have_fun') %></p>
```
Comment (bakoe) [inline app/views/team_mailer/team_registered_to_event.html.erb:5]: Nice find that we can use `.have_fun` without explicitly defining e. g. `user_added_to_team` as the locale to look into for the localization. 👍

**Review by bakoe — APPROVED** [review]
LGTM! 👍 I have also confirmed that the change is working by using `rails c` to provoke both e-mail notification events – the localizations are used as one would expect.

**Review by bakoe — CHANGES_REQUESTED** [review]
Although not explicitly defined in the initial user story #308 or in @svitlanakalancha's change request stated in Slack (see channel #307-308-coordination), I'm pretty sure that @svitlanakalancha especially wanted that User's don't get the ”Viel Erfolg beim Training!“ message when matches are canceled and when events are canceled/finished. Could you maybe integrate this in this PR as well @conradlempert?

**Comment by bakoe** [comment]
@conradlempert See the following message from the Slack channel for explanation about what I meant regarding @svitlanakalancha's change request:

ohne titel 3

Maybe just remove the `have_fun` part from the notifications about the following events?

- match being canceled
- event being canceled
- event being finished

**Review by bakoe — APPROVED** [review]

**Pull request merged**